bookwyrm/templates/lists/list_items.html

```diff
@@ -47,4 +47,7 @@ <h4 class="card-header-title is-clipped">
         </div>
     </div>
     {% endfor %}
+    {% if not lists or not lists.exists %}
```

Comment:
`lists.exists` is always false so even when a list is found it still says "No lists found".

Consider using `{% empty %}` instead, above `{% endfor %}`

Reply:
I was genuinely thinking that I wished something like `{% empty %}` existed when I wrote this 🤦. Amazing, thank you